verification.py

```diff
@@ -5919,6 +5919,63 @@ def test_bottleneck_coalescence_time_T2(self):
         self._run(2)
 
 
+class MicrosatAnalytical(Test):
+    """
+    Analytical comparisons wrt to microsatellites
+    """
+
+    def test_SMM_variance(self):
+        """
+        Under the strict SMM model we expect the
+        Variance in repeat number of a sample
+        to be equal to 4Nu. This result has
+        been derived a number of times but a good
+        citation is Goldstein et al (1995)
+        https://doi.org/10.1093%2Fgenetics%2F139.1.463
+        """
+
+        def copy_number_matrix(ts):
```

Comment:
No need to make this a private function, it might be useful for other tests. Just put it at the top level.